tests/TestDataService/OrderCreation.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect, type Order, APIResponse } from '../../src/index';
+
+
+test('Order creation with TestDataService', async ({
```

Comment:
I've moved this into it's own test, because the spec file got way to big. We should not write tests that are bigger than 100 lines of code.